deny.toml

```diff
@@ -12,7 +12,8 @@
 multiple-versions = "deny"
 
 # Don't allow wildcard ("any version") dependencies
-wildcards = "deny"
+# TODO: Change to `deny` after we upgrade to crates.io version of the ECC dependencies (#8749).
+wildcards = "allow"
```

Comment:
Why do we need this?

Reply:
When we do for example `zcash_address = { git = "https://github.com/zcash/librustzcash/", branch = "main"}` we are actually using a wildcard that will match any version `main` has last. `cargo-deny` will fail if we don't make this change.